synopsys-task/src/synopsys-task/application-constant.ts

```diff
@@ -22,17 +22,17 @@ export const POLARIS_ASSESSMENT_TYPES_KEY = "bridge_polaris_assessment_types";
 export const POLARIS_SERVER_URL_KEY = "bridge_polaris_serverUrl";
 
 // Coverity
-export const COVERITY_URL_KEY = "bridge_coverity_url";
-export const COVERITY_USER_KEY = "bridge_coverity_user";
-export const COVERITY_PASSPHRASE_KEY = "bridge_coverity_passphrase";
+export const COVERITY_URL_KEY = "bridge_coverity_connect_url";
 export const COVERITY_USER_NAME_KEY = "bridge_coverity_connect_user_name";
+export const COVERITY_PASSPHRASE_KEY = "bridge_coverity_passphrase";
 export const COVERITY_USER_PASSWORD_KEY =
```

Comment:
Why we have two passphrase related key. Please check, if we are using proper keys which is getting used for coverity.